pubsub/azure/servicebus/subscription.go

```diff
@@ -203,11 +202,7 @@ func (s *subscription) tryRenewLocks() {
 func (s *subscription) receiveMessage(ctx context.Context, handler azservicebus.HandlerFunc) error {
 	s.logger.Debugf("Waiting to receive message on topic %s", s.topic)
 	if err := s.entity.ReceiveOne(ctx, handler); err != nil {
-		if strings.Contains(err.Error(), "force detached") {
-			return nil
-		}
-
-		return fmt.Errorf("%s error receiving message on topic %s, %s", errorMessagePrefix, s.topic, err)
+		return fmt.Errorf("%s error receiving message on topic %s, %w", errorMessagePrefix, s.topic, err)
```

Comment:
Using `%w` is required to wrap the underlying AMQP error so it can be detected by `errors.Is`/`errors.As`.

https://blog.golang.org/go1.13-errors